Detectors/TRD/workflow/src/TRDTrackletTransformerSpec.cxx

```diff
@@ -35,6 +35,13 @@ void TRDTrackletTransformerSpec::run(o2::framework::ProcessingContext& pc)
 {
   LOG(INFO) << "Running tracklet transformer";
 
+  TrackletTransformer transformer(mTimestamp);
+
+  if (!transformer.hasCalibration()) {
+    // ccdb object was not found for specified timestamp
```

Comment:
Would be got to throw an error message here

Reply:
The error gets thrown at line 57 in `TrackletTransformer::loadCalibrationParameters()`.